gridstatus/base.py

```diff
@@ -26,11 +26,25 @@ class Markets(Enum):
     """Names of LMP Markets"""
 
     REAL_TIME_5_MIN = "REAL_TIME_5_MIN"
+
+    REAL_TIME_5_MIN_EX_ANTE = "REAL_TIME_5_MIN_EX_ANTE"
+
+    REAL_TIME_5_MIN_EX_POST_PRELIM = "REAL_TIME_5_MIN_EX_POST_PRELIM"
+    REAL_TIME_5_MIN_EX_POST_FINAL = "REAL_TIME_5_MIN_EX_POST_FINAL"
     REAL_TIME_5_MIN_FINAL = "REAL_TIME_5_MIN_FINAL"
+
     REAL_TIME_15_MIN = "REAL_TIME_15_MIN"
     REAL_TIME_HOURLY = "REAL_TIME_HOURLY"
+
+    REAL_TIME_HOURLY_EX_POST_PRELIM = "REAL_TIME_HOURLY_EX_POST_PRELIM"
```

Comment:
should we update `MISO` to correctly use these new market names rather than `Markets.REAL_TIME_HOURLY_FINAL`, `Markets.REAL_TIME_HOURLY_PRELIM`, `Markets.DAY_AHEAD_HOURLY`? 

perhaps we also consider making a breaking change downstream in our api to just get the "market" column and the dataset names correct

Reply:
> should we update MISO to correctly use these new market names

Based on our discussion, it sounds like no.